app/javascript/dashboard/i18n/locale/en/integrations.json

```
@@ -81,6 +84,51 @@
     },
     "CONNECT": {
       "BUTTON_TEXT": "Connect"
+    },
+    "DASHBOARD_APPS": {
+      "HEADER_BTN_TXT": "Add new dashboard app",
+      "SIDEBAR_TXT": "<p><b>Dashboard Apps</b> </p> <p>Dashboard apps allow you to create custom pages(apps) right inside your conversations page in chatwoot dashboard. Chatwoot sends the current conversation context(a JSON payload) which contains the user datails and conversation as a window event, which you can use to create your own pages.</p><p>You can read more about dashboard apps here and the demo here in the <a href='#'>docs</a></p>",
+      "LIST": {
+        "404": "There are no dashboard apps configured yet",
+        "LOADING": "Fetching dashboard apps",
+        "TABLE_HEADER": [
+          "Name",
+          "Type",
+          "URL"
+        ],
+        "EDIT_TOOLTIP": "Edit app",
+        "DELETE_TOOLTIP": "Delete app"
+      },
+      "FORM": {
+        "TITLE_LABEL": "Dashboard app name",
```

```suggestion
        "TITLE_LABEL": "Name",
```

Let us keep it simple.